### PR TITLE
Add more text to file

### PR DESCRIPTION
**Pull request by rpoole444**
Hey I don't really like waffles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 supppp
 
-waffles
\ No newline at end of file
+waffles
+Cupcake for breakfast!
```
Comment (keenans1) [inline README.md]: 😡

Comment (rpoole444) [reply README.md]: 🎺

**Review by keenans1 — CHANGES_REQUESTED** [review]
update cupcakes for breakfast to waffles all day

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 supppp
 
 waffles
-Cupcake for breakfast!
\ No newline at end of file
+okay okay waffles all day
```
Comment (rpoole444) [inline README.md:4]: here ya go

**Review by keenans1 — APPROVED** [review]
looks good!

**Pull request merged**